### DISCUSSION TITLE
How to disable formatting errors?

### DISCUSSION THREAD

**Discussion by RyannDaGreat** [w0rp/ale#4042] [Q&A - Ask for help with problems]
I'm coding in python and getting Flake8 errors I don't care about such as "E266: too many leading '#' for block comment"
How do I disable these annoying formatting errors while keeping more important errors such as forgetting parenthesis?

**Comment by w0rp** [comment]
You can disable them in your flake8 configuration files, or with line comments. https://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html

**Reply by RyannDaGreat** [reply]
Thank you - this will probably work, but this would make it system-wide right? Is there a way to do it from the .vimrc file so that it only changes the flake8 settings for vim?

**Reply by hsanson** [reply]
You can set options for ALE to use when calling flake8 with `g:ale_python_flake8_options`. This would affect only vim.

For example I see flake8 has an option `--append-config` that should allow you to create some config file and then tell ALE to use it when running flake8.

**Reply by RyannDaGreat** [reply]
Thank you, could you please give a simple example of how it should look in my .vimrc? 

Would an example be like this?: 
```
let g:ale_python_flake8_options='--ignore=E101,E111,E112,E113,E114'
```

**Reply by hsanson** [reply]
Yes, that seems to work fine.  Tested it on one of my projects.
